api/app/Policies/PoolPolicy.php

```diff
@@ -193,4 +193,21 @@ public function archiveAndUnarchive(User $user, Pool $pool)
 
         return $user->isAbleTo('archive-team-pool', $pool->team);
     }
+
+    /**
+     * Determine whether the user can view pool assessments
+     *
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function viewAssessments(User $user, Pool $pool)
+    {
+        // request responder or platform admin view all
+        if ($user->hasRole('request_responder') || $user->hasRole('platform_admin')) {
```

Comment:
Check permissions, don't check roles directly. If that means a new permission has to be created, that's okay.

Reply:
Adjusted policy in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8022/commits/c82152a786f472938d76894c2d0f30b819a824fb